changedetectionio/forms.py

```diff
@@ -1,13 +1,30 @@
-from wtforms import Form, SelectField, RadioField, BooleanField, StringField, PasswordField, validators, IntegerField, fields, TextAreaField, \
-    Field
+import re
 
-from wtforms import widgets, SubmitField
-from wtforms.validators import ValidationError
+from wtforms import (
```

Comment:
thanks for cleaning this stuff up

Reply:
This is the work of `isort` (https://github.com/PyCQA/isort) running automatically on every save!
It may make sense to run it (and perhaps also some code formatter) across the whole project and mandate it in the contributing file, that would make contributing and keeping the codebase uniform easier.

Reply:
@timlod I made a wiki here https://github.com/dgtlmoon/changedetection.io/wiki/Developer-Notes could you update this with some of the very cool tools you're using? I'm finding it hard to keep up :)